ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/monitor-sessions.yml

```diff
@@ -0,0 +1,38 @@
+---
+monitor_sessions:
+  - name: myMonitoringSession1
+    sources:
+      - name: Ethernet5
+        direction: both
+        access_group:
+          type: ip
+          name: ipv4ACL
+          priority: 10
+      - name: Ethernet0
+        access_group:
+          type: ipv6
+          name: ipv6ACL
+    destinations:
+      - Ethernet48
+    encapsulation_gre_metadata_tx: true
+    header_remove_size: 32
+    truncate:
+      enabled: true
+  - name: myMonitoringSession2
+    sources:
+      - name: Ethernet12
+        direction: rx
+      - name: Ethernet18
```

Comment:
Not a suggestion but provide an example for interface_range if possible. So that we cover that test as well.